src/ui-components/BalanceInput.tsx

```diff
@@ -58,24 +57,55 @@ const BalanceInput = ({ className, label = '', helpText = '', onChange, placehol
 	// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
-	return <div className={`${className} w-full flex flex-col`}>
-		<label className='mb-[2px] flex items-center text-sm'>
-			{label} {helpText && <HelperTooltip className='ml-2' text={helpText}/> }
+	return <div className={`${className} w-full flex flex-col balance-input`}>
+		<label className='mb-[2px] inner-headings'>
+			{label}
 			{address && withBalance && <span><Balance address={address} onChange={onAccountBalanceChange} /></span>
 			}
 		</label>
 		<Form.Item
-			name="balance"
+			name={formItemName}
 			initialValue={balance ? Number(formatedBalance(balance.toString(), unit)) : ''}
-			rules={noRules ? []: [
+			rules={noRules ? [ {
```

Comment:
try and remove noRules

Reply:
Please remove noRules prop

Reply:
y abhi search ke sath I will try to remove